oc_cdtapi/DmsGetverAPI.py

```diff
@@ -46,6 +93,92 @@ def create_distr_request(self, version=None, source_version=None, distr_type=Non
 
         return distr_state_info
 
+    def download_file(self, image_id):
+        """
+        Downloads specified image to file
+        :param str image_id: id of image to download
+        :return: path to downloaded file
+        """
+        logging.debug('Reached download_file')
+        logging.debug('image_id: [%s]' % image_id)
+        logging.debug('Unsetting auth')
+        self.web.auth = None
+        headers = self.get_headers()
+        url = posixpath.join('api', 'v1', 'images', image_id, 'download')
+        logging.debug('url: [%s]' % url)
+        tf = tempfile.NamedTemporaryFile(delete=False)
+        tfn = tf.name
+        logging.debug('tf: [%s]' % tfn)
+        r = self.get(url, headers=headers, stream=True)
+        for chunk in r.iter_content(chunk_size=8192):
+            tf.write(chunk)
+        tf.seek(0)
+        return tfn
+
+    def get_audit(self, audit_id):
+        """
+        Requests audit record by record id
+        :param str audit_id: id of audit record
+        :return: audit record
+        """
+        logging.debug('Reached get_audit')
+        logging.debug('audit_id: [%s]' % audit_id)
+        self.web.auth = None
+        self.raise_exception_high = 499
```

Comment:
Just the same were done during *create_custom_image* and not restored back.
It is bad idea since we are providing credentials during class instantiation, and we are loosing them here.
Any other method which require synchronization will stop working.